### DISCUSSION TITLE
docusaurus integration

### DISCUSSION THREAD

**Discussion by obgibson** [code-hike/codehike#128] [Q&A] [answered]
Has anyone had any experience integrating codehike with docusaurus?

I have added the following to `docusaurus.config.js`

```
  presets: [
    [
      'classic',
      /** @type {import('@docusaurus/preset-classic').Options} */
      ({
        docs: {
          remarkPlugins: [[remarkCodeHike, { theme }]],
```

My code blocks now just show the code as a raw string, rather than a purty box with highlighting.

Are there other steps to integrate?

**Comment by pomber** [comment]
Code Hike needs MDX v2 which docusaurus doesn't support right now. Once it is supported, I'll make a demo project and make sure everything works.

**Reply by pomber** [reply]
By the way, I created issue #133 to avoid similar confusion.

**Comment by obgibson** [comment]
I see. Are there any documentation platforms that already use v2 that you'd recommend plugging code hike into?

**Reply by pomber** [reply]
[Storybook](https://storybook.js.org/docs/react/writing-docs/introduction) (it can also be used for docs) should work. [Nextra](https://twitter.com/shuding_/status/1478877774081318920) was migrating to mdx v2, not sure what's the status right now. 
The other option is a more general-purpose framework, you could use Next.js or Remix.

**Reply by pomber** [reply]
@cabljac was working on something too, see #113

**Reply by cabljac** [reply]
Yup we use xdm, which supports codehike, and codehike will be supported in the next release of [docs.page](https://docs.page/homepage) under a flag.  We're almost ready!

**Comment by pomber** [comment, marked as answer]
Docusaurus is now supported https://codehike.org/docs/installation/docusaurus
